deploy_scripts/start.sh

```diff
@@ -34,11 +35,12 @@ sudo cp -r /home/ec2-user/flashcrow/dist /usr/share/nginx/html/flashcrow
 # update database
 . /home/ec2-user/psqlArgs.config
 # shellcheck disable=SC2154
-/home/ec2-user/flashcrow/scripts/db/db-update.sh --psqlArgs "$psqlArgs"
+/var/app/flashcrow/scripts/db/db-update.sh --psqlArgs "$psqlArgs"
 
 # start flashcrow
+cd /var/app/flashcrow
 # shellcheck disable=SC2046
-env $(xargs < /home/ec2-user/cot-env.config) NODE_ENV=production NODE_EXTRA_CA_CERTS=/home/ec2-user/flashcrow/ssl/extra-ca-certs.cer forever start /home/ec2-user/forever.json
+sudo -u appsvc env $(xargs < /var/app/config/cot-env.config) PATH="$PATH" NODE_ENV=production NODE_EXTRA_CA_CERTS=/var/app/ssl/extra-ca-certs.cer forever start /var/app/flashcrow/forever.json
```

Comment:
here: we can set `forever`'s config file via that last argument, which means we don't need to re-copy `forever.json` at all!